### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by magnologan**
Add reference to SUNSPOT - https://www.crowdstrike.com/blog/sunspot-malware-technical-analysis/

Slack discussion: https://cloud-native.slack.com/archives/CDJ7MLT8S/p1610478830312800

### CODE REVIEW

**Comment by lumjjb** [comment]
It looks like they are referencing the same page, can we have this link be under another link instead of a distinct row, unless i'm missing something here. 

`
[1](https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html),[2](https://msrc-blog.microsoft.com/2020/12/13/customer-guidance-on-recent-nation-state-cyber-attacks/),[3](New link)`

**Comment by magnologan** [comment]
It's kind of a different payload, that's why they called it differently. It is related to SolarWinds but another variant. But no problem, I'll keep them on the same row.

File: supply-chain-security/compromises/README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ We welcome additions to this catalog by
 
 | Name              | Year               | Type of compromise    | Link        |
 | ----------------- | ------------------ | ------------------    | ----------- |
+| [SUNSPOT/Solarigate](2020/solarwinds.md) | 2020 | Malware | [1](https://www.crowdstrike.com/blog/sunspot-malware-technical-analysis/) |
```
Comment (TheFoxAtWork) [inline supply-chain-security/compromises/README.md:18]:
```suggestion
| [SUNSPOT/Solarigate](2020/solarwinds.md) | 2020 | Publishing Infrastructure | [1](https://www.crowdstrike.com/blog/sunspot-malware-technical-analysis/) |
```

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
Need to remain consistent with the naming schema for supply chain attacks

**Pull request closed without merging**